### DISCUSSION TITLE
Σε σχέση με 2ο αίτημα ενσωμάτωσης

### DISCUSSION THREAD

**Discussion by Icelandan** [ioniodi/sitegr#191] [Q&A] [answered]
Έχω παρατηρήσει πως σε 1-2 issues που άνοιξαν συμφοιτητές μου σε σχέση με τη δημιουργία νέας σελίδας για το site έχουν γίνει σχόλια ότι δεν είναι ισάξια του παραδοτέου ('good first issue' tag).

Παρόλα αυτά έχω διαβάσει τις οδηγίες, και λένε ξεκάθαρα: <<Για τα θέματα ενδιάμεσης δυσκολίας στο **2ο-3ο αίτημα ενσωμάτωσης** το ζητούμενο είναι μια συνεισφορά που δείχνει μεγαλύτερη κατανόηση του συστήματος (π.χ., βελτίωση της τεκμηρίωσης ή της διαδικασίας συνεισφοράς, **προσθήκη νέων σελίδων**, βελτιώσεις στον κώδικα λίκουηντ, κτλ).>>

Εφόσον το 'προσθήκη νέων σελίδων' συμπεριλαμβάνετε στα παραπάνω δεν καταλαβαίνω ποιο είναι το πρόβλημα με τέτοια issues;

**Comment by epidrome** [comment, marked as answer]
νέες σελίδες εννοούμε σελίδες όπως αυτή, 
https://epic-hamilton-da9ac8.netlify.app/humanistic

δλδ όχι απλά προσθήκη περιεχομένου, αλλά κυρίως προσθήκη οπτικής-φίλτρου πάνω στα υπάρχοντα δεδομένα

οι οδηγίες ενημερώθηκαν ώστε να μην υπάρχει παρανόηση

πολύ καλή απορία, ευχαριστούμε!
